Gordon360/Exceptions/CustomExceptions.cs

```diff
@@ -21,4 +21,9 @@ public class UnprocessibleEntity : Exception
     {
         public string ExceptionMessage { get; set; }
     }
+    public class NotFound : Exception
+    {
+        public string ExceptionMessage { get; set; }
+    }
```

Comment:
This already exists, it's the `ResourceNotFound` Custom Exception.

Reply:
I'll make that change, didn't realize. BUT in retrospect, it should have been obvious considering it is relatively common